modin/pandas/test/test_general.py

```diff
@@ -801,3 +802,31 @@ def test_empty_dataframe():
 def test_empty_series():
     s = pd.Series([])
     pd.to_numeric(s)
+
+
+@pytest.mark.parametrize(
+    "arg",
+    [[1, 2], ["a"], 1, "a"],
+    ids=["list_of_ints", "list_of_invalid_strings", "scalar", "invalid_scalar"],
+)
+def test_to_timedelta(arg):
+    # This test case comes from
+    # https://github.com/modin-project/modin/issues/4966
+    eval_general(pd, pandas, lambda lib: lib.to_timedelta(arg))
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+def test_series_to_timedelta(data):
+    if isinstance(data, dict):
+        modin_series = pd.Series(data[next(iter(data.keys()))])
+        pandas_series = pandas.Series(data[next(iter(data.keys()))])
+    else:
+        modin_series = pd.Series(data)
+        pandas_series = pandas.Series(data)
+
+    modin_series = pd.to_timedelta(modin_series)
+    pandas_series = pandas.to_timedelta(pandas_series)
+    df_equals(
+        modin_series.to_frame(name="timedelta"),
+        pandas_series.to_frame(name="timedelta"),
+    )
```

Comment:
let's use `eval_general()` here, too - it takes care of comparing exceptions if they raise (plus it correctly waits for async Modin calls as they only raise exceptions when we ask for the result):
```suggestion
    def make_frame(lib):
        series = lib.Series(
            next(iter(data.values())) if isinstance(data, dict) else data
        )
        return lib.to_timedelta(series).to_frame(name="timedelta")

    eval_general(pd, pandas, make_frame)

```